File: 프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

vector<int> solution(vector<int> num_list) {
    vector<int> answer(2,0);
    for(auto x: num_list) {
        if (x % 2 == 0) {
            answer[0] += 1;
        } else {
            answer[1] += 1;
        }
    }
    return answer;
}

File: 프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.py
def solution(num_list):
    answer = []
    even = 0
    od = 0
    for n in num_list:
        if n % 2 == 0:
            even += 1
        else:
            od += 1
    answer.append(even)
    answer.append(od)
    return answer